### PR TITLE
CPDT-203 Update style of b-input and b-textarea

### PR DESCRIPTION
**Pull request by eleanormal**
## Type of change
- [ ] Bug Fix
- [x] New Feature
- [ ] Refactor
- [ ] Project Build
- [ ] Project Configuration

## Submission Checklist

- [x] Have you followed the guidelines in our [Contributing document](https://github.com/fanappics/buoy/blob/master/CONTRIBUTING.md)?
- [ ] Did you create appropriate tests?

## Description
Update input element and label style to match new style conventions.
Moved things around in main.styl so that associated element styles are near one another

Question about css color naming conventions:
The main purpose of variable names for colors is so that you don't have to go to each place that the color is used to change it when doing theming. If we name the colors with a visually descriptive name ($night-rider, $cerulean, etc) then we'd have to have different stylesheets for each theme. If, however, we give them a functionally descriptive name ($input-border, $focus, etc), we can just have different variable `.styl` docs that we pull in for different themes.

I included both in this PR so as not to totally rock the boat, but I would like input on this issue.

## Related Issues / Bugs / Feature Requests
https://jira.fanatics.com/browse/CPDT-203

## Test Deployment URL
https://rawgit.com/fanappics/buoy/ft-cpdt-203/docs/index.html#binput

## Deployment Notes

N/A

## Steps to Test or Reproduce

Go to the styleguide link above 
Observe that it matches the styles of input fields in https://app.zeplin.io/project/5a7a2d4c69fae4776dce32dd/screen/5a7a2dc78c79ac0755fb83a9

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -8,5 +8,5 @@
     <link href="main.css" rel="stylesheet"></head>
     <body>
 	<div id="app"></div>
-    <script type="text/javascript" src="build/bundle.2753c2dd.js"></script></body>
+    <script type="text/javascript" src="build/bundle.0856d193.js"></script></body>
```
Comment (ghost-) [inline docs/index.html]: This file isn't there.  Maybe a rebuild, or build folder not checked in?

Check the styleguide here -

https://rawgit.com/fanappics/buoy/ft-cpdt-203/docs/index.html

File: src/components/BInput/BInput.vue [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="flex">
+  <div class="flex column">
```
Comment (mlear) [inline src/components/BInput/BInput.vue]: We may want to change the class name from column to flex-column or something else.  Column is really generic.

Comment (eleanormal) [reply src/components/BInput/BInput.vue]: I like that idea. I think we do need a general naming convention for our classes. @stevenmwade suggested `b-*` and then most people didn't respond to the poll

File: docs/index.html (hunk 1 above)
Comment (eleanormal) [reply docs/index.html]: Weird. I'll check that out.

**Comment by eleanormal** [comment]
Started prefixing global styles with `b-`. Have left backwards compatibility so as not to have to change lots of unrelated files for this PR but I'll get them each as those files are touched as part of `CPDT-189`

**Comment by kierachell** [comment]
- Since this is a style thing:
screen shot 2018-02-16 at 7 22 40 am
screen shot 2018-02-16 at 7 22 44 am


- What's the difference, visually, if I don't look at the labels? Maybe a question to @cochoa89

**Comment by eleanormal** [comment]
@kierachell Visually the only difference is textarea is bigger

**Pull request merged**